src/dug/server.py

```diff
@@ -147,6 +147,16 @@ async def search_program( program_name: Optional[str] = None):
         "status": "success"
     }
 
-
+@APP.post('/program_list')
```

Comment:
there's an endpoint for this already (https://github.com/helxplatform/dug/blob/master/src/dug/core/async_search.py#L94) , do we want to use that instead?

Reply:
yes